elasticsearch/sgconfig/sg_roles.yml

```diff
@@ -49,6 +49,18 @@ sg_role_admin:
   cluster:
     - CLUSTER_ALL
 
+sg_role_jaeger:
+  cluster:
+    - indices:data/write/bulk
+    - CLUSTER_MONITOR
+  indices:
+    '*':
```

Comment:
can we make this `'?jaeger-*'` this means that indices would be `.jaeger-*`

Reply:
we use indices:
*`jaeger-span-2019-01-29`
* `jaeger-service-2019-01-29`
* `jaeger-span-read`
* `jaeger-span-write`
* `jaeger-service-read`
* `jaeger-service-write`
* `jaeger-span-archive`
* `jaeger-span-archive-read`
* `jaeger-span-archive-write`

 and all of these indices can be prefixed with arbitrary prefix `*-` or `*:` e.g. `.-foo-`/`foo:`